App/Levels/wave.js

```diff
@@ -197,14 +197,32 @@ const WAVE = [
     ],
     sledders: [
       {
-        asset: 'images.sam_sled',
+        asset: 'images.lunchbox_sled',
         x: 0,
         y: 0,
+        speech: [
+          {
+            x: 0.4,
+            y: 0.70,
+            content: 'Oh wow it\'s toasty here',
+            direction: 'up',
+            distance: 1.2,
+          },
+        ],
       },
       {
-        asset: 'images.lunchbox_sled',
+        asset: 'images.sam_sled',
         x: TAU,
         y: 0,
+        speech: [
+          {
+            x: 0.8,
+            y: 0.9,
+            content: 'Why are you cold?',
```

Comment:
Wait, I don't understand this line. Why is one asking the other why they are cold?

Reply:
> Wait, I don't understand this line. Why is one asking the other why they are cold?

Oh I meant it a bit sarcastically since ghosts can't feel the cold/heat

Reply:
But the other one says they are toasty

Reply:
yea now that I think about it, it doesn't make much sense
any ideas?

Reply:
What if Jack is like "Wow, it's toasty here!" and then Ada says "Take your scarf off, Jack."

Reply:
great!